### PR TITLE
Add portable-simd repo under automation

### PR DESCRIPTION
**Pull request by Kobzol**
Repo: https://github.com/rust-lang/portable-simd

CC @calebzulawski, mainly about the branch protection.

Extracted from GH:
```
org = "rust-lang"
name = "portable-simd"
description = "The testing ground for the future of portable SIMD in Rust"
bots = []

[access.teams]
libs = "write"
libs-contributors = "write"
project-portable-simd = "write"
security = "pull"
libs-api = "write"

[access.individuals]
badboy = "admin"
kennytm = "write"
programmerjake = "write"
workingjubilee = "write"
rust-lang-owner = "admin"
pietroalbini = "admin"
dtolnay = "write"
cramertj = "write"
thomcc = "write"
cuviper = "write"
the8472 = "write"
jdno = "admin"
rylev = "admin"
BurntSushi = "write"
yaahc = "write"
SimonSapin = "write"
ChrisDenton = "write"
sunfishcode = "write"
miguelraz = "write"
joshtriplett = "write"
Amanieu = "write"
m-ou-se = "write"
JohnTitor = "write"
Mark-Simulacrum = "admin"
KodrAus = "write"
calebzulawski = "write"
```

### CODE REVIEW

**Review by rylev — APPROVED** [review]

**Comment by calebzulawski** [comment]
What's the consequence of this change on branch protection? Thanks!

**Comment by Kobzol** [comment]
Now it is required to do a PR to merge something into `master`, and each PR must have at least one approval (from someone else than the PR author). Also, the `x86-tests` CI job has to succeed before a PR can be merged.

Let me know if you want to remove the branch protection. Or we can make it "weaker", e.g. by disabling the reliance on the CI job, or to remove the requirement that at least one approval is needed.

**Comment by calebzulawski** [comment]
As much as I'd like to do a little better, I think it would be best to not require CI to pass, and not require an additional approval, at least for now.  We didn't even require PRs in the past, but I think that's probably a good change.

Also, where did all those additional individual permissions come from?  At least some overlap with the teams. If possible, I think it would make sense to limit it to just the appropriate teams.  All of the regular contributors are part of project-portable-simd.

**Pull request merged**
